xcode/Subconscious/Shared/Components/Common/Buttons/PillButtonStyle.swift

```diff
@@ -39,7 +39,7 @@ struct PillButtonView: View {
                 case .regular:
                     return AppTheme.unit * 12
                 case .small:
-                    return AppTheme.unit * 9
+                    return AppTheme.unit * 6
```

Comment:
Minimum hit target size is 44pt. The standard iOS border button style is visually 33pt high, but, testing, I can see it has generous invisible padding at the top and bottom to bring the hit target to something around 44pt.

Would suggest we stay around 32pt high for visual consistency with the rest of the OS (I often round to the nearest number divisible by 8 or 4), and add invisible overlay or padding to bring to 44pt.

Reply:
Good call, I've updated the visual height to 32pt (AppTheme.unit * 8) and given the button itself a minimum tap target height of 44pt.

<img width="376" alt="image" src="https://github.com/subconsciousnetwork/subconscious/assets/5009316/a1c075af-d8f4-4c97-a638-a6569b8cd15e">